src/components/layout.module.scss

```diff
@@ -0,0 +1 @@
+
```

Comment:
if this file is not needed, then it might be cleaner to remove it. also its import reference from the JS file.